src/reducer.js

```diff
@@ -90,6 +90,8 @@ export default function undoable (reducer, rawConfig = {}) {
     neverSkipReducer: false,
     ignoreInitialState: false,
     syncFilter: false,
+    extension: () => (state) => state,
+    disableWarnings: false,
```

Comment:
this could be an argument to the extension function instead of a redux-undo wide config

Reply:
We can do that.